conf/application.conf

```diff
@@ -315,6 +315,9 @@ feature {
   override-start-and-end-dates-for-news-items {
     enabled = false
   }
+  pega{
+   enabled = false
```

Comment:
Maybe rename to interstitialPageEnabled - since I think we will have further pega features in future

Reply:
Good idea